src/PowerSystems.jl

```diff
@@ -195,6 +195,11 @@ export VoltageModeControl
 export CurrentModeControl
 export RECurrentControlB
 
+# InverterLimiters Export
+export MagnitudeCurrentLimiter
```

Comment:
We missing the circle one?

Reply:
Magnitude is circle, Instantaneous is Square, based on this paper: https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=9973369

We can change the name to circular, and square